README.md

```diff
@@ -42,27 +42,29 @@
 [Contributed packages]: https://github.com/fused-effects/fused-effects#contributed-packages
 [Comparison to `mtl`]: https://github.com/fused-effects/fused-effects#comparison-to-mtl
 [Comparison to `freer-simple`]: https://github.com/fused-effects/fused-effects#comparison-to-freer-simple
+[Comparison to `polysemy`]: https://github.com/fused-effects/fused-effects#comparison-to-polysemy
 
 
 ## Overview
 
-`fused-effects` is an effect system for Haskell emphasizing expressivity and efficiency. The former is achieved by encoding [algebraic](#algebraic-effects), [higher-order](#higher-order-effects) effects, while the latter is the result of [fusing](#fusion) effect handlers all the way through computations.
-
-Readers already familiar with effect systems may wish to start with the [usage](#usage) instead.
+`fused-effects` is an [effect system](https://en.wikipedia.org/wiki/Effect_system) for Haskell that values expressivity, efficiency, and rigor. It provides a framework for encoding [algebraic](#algebraic-effects), [higher-order](#higher-order-effects) effects, includes a library of the most common effect types, and generates efficient code by [fusing](#fusion) effect handlers through computations. It was developed as a part of the [`semantic`](https://github.com/github/semantic) program analysis toolkit, but is suitable for general use in hobbyist, research, or industrial contexts.
```

Comment:
NB: it actually wasn’t developed as a part of `semantic`. I developed it in isolation, on my own time.

Reply:
Changed to "is used extensively".